playwright/tests/ledger.spec.ts

```diff
@@ -6,7 +6,7 @@ test.describe('Ledger', () => {
     expect((await page.request.head('/')).headers()).toHaveProperty('permissions-policy')
     await expectNoErrorsInConsole(page)
 
-    await page.goto('/open-wallet/ledger')
```

Comment:
Playwright test should keep user flow. let's bring back this line -> find and click button -> expect /usb to be loaded

Reply:
Keeping as is, due to this error in CI:
![image](https://github.com/oasisprotocol/oasis-wallet-web/assets/9722540/aa244217-2096-4894-b593-6f404eb3d91c)

And I don't want to implementy a hacky way to mock Bluetooth functionality in E2E tests.